test.js

```diff
@@ -19,4 +19,7 @@ test('main', t => {
 	t.is(cliTruncate('unicorn', 6, {position: 'start'}), '…icorn');
 	t.is(cliTruncate('unicorn', 5, {position: 'middle'}), 'un…rn');
 	t.is(cliTruncate('unicorns', 6, {position: 'middle'}), 'uni…ns');
+	t.is(cliTruncate('unicorns', 5, {position: 'end', space: true}), 'uni …');
```

Comment:
These should be in a separate test named `space option`. I'd also like to see many more assertions with different kinds of sentences.

Reply:
I have added a few more tests, let me know if we need more.